docs/2.7.0/docs/daml-finance/concepts/contingent-claims.rst

```diff
@@ -203,7 +203,19 @@ The Observation Parameter
 =========================
 
 ``o`` is the type used to represent ``Observation``\ s, which are time-dependent quantities that can
-be observed at any given time (such as the “EURUSD” exchange rate in the example above).
+be observed at any given time (such as the “EURUSD” exchange rate in the example above). There are
```

Comment:
This section is meant to describe the type parameters involved in a claim, so the description of the specific observation combinators should probably live in a separate place.

There are also other combinators in addition to `Observe` and `ObserveAt`, such as `Const` and the algebraic combinators `+` `*`

Reply:
> so the description of the specific observation combinators should probably live in a separate place.

Where do think this belongs? In one of the existing places or in a new one?